folly/container/test/F14MapTest.cpp

```diff
@@ -1777,6 +1777,26 @@ TEST(F14Map, containsWithPrecomputedHash) {
   testContainsWithPrecomputedHash<F14FastMap>();
 }
 
+template<template<class...> class TMap>
+void testEraseIf()
+{
```

Comment:
And this to:

```
template <template <class...> class TMap>
void testEraseIf() {
```

Reply:
No problem. Sorry for the old style habits; I just let clang-format do its thing on my code to not worry about these things too much.